deploy/addon/pulsar.md

```diff
@@ -0,0 +1,413 @@
+Unique:        17dfec62b62da36562a0998d496bb3aa30f229138ec810a070084bdf1c55be3a
+
+==========
+```
+
+* The authority block is the cluster authentication block (the cluster admin Biscuit token).
+* The block n°1 is an attenuation of the authority block to only authorize operations on `tenant = "user_1235678-f54e-4e09-848c-1953af6e3e89"` and `namespace = "pulsar_1235678-6b36-4af2-be1f-d97862c0c41c"`.
+
+Attenuate it:
+
+```bash
+biscuit attenuate addon.biscuit
+```
+
+This will open your `$EDITOR` to type the attenuation.
+
+Put
+
+```bash
+check if topic_operation(#ambient, "user_12345678-f54e-4e09-848c-1953af6e3e89", "pulsar_12345678-6b36-4af2-be1f-d97862c0c41c", $topic, $operation), $topic.starts_with("my-own-prefix")
+```
+
+Then it outputs the attenuated token. Inspect it to ensure your attenuation:
+
+```bash
+Authority block:
+== Datalog ==
+right(#authority, #admin);
+
+== Revocation ids ==
+Content-based: fe7526a27b43fa7d5386c31b3efb6c53b551b4a8e3e969f4dc074497b3942a57
+Unique:        faf6cab8a4dffad77633181d6f924414980bb6d76c5298b88f700c11659c7407
+
+==========
+
+Block n°1:
+== Datalog ==
+check if topic_operation(#ambient, "user_12345678-f54e-4e09-848c-1953af6e3e89", "pulsar_50ee8f69-6b36-4af2-be1f-d97862c0c41c", $2, $3) or namespace_operation(#ambient, "user_12345678-f54e-4e09-848c-1953af6e3e89", "pulsar_12345678-6b36-4af2-be1f-d97862c0c41c", $2);
+
+== Revocation ids ==
+Content-based: a35a92a278a3ad9ec5db8dc3e905cfbc17f5a8e2cb13ff39b69b003500fe6c46
+Unique:        17dfec62b62da36562a0998d496bb3aa30f229138ec810a070084bdf1c55be3a
+
+==========
+
+Block n°2:
+== Datalog ==
+check if topic_operation(#ambient, "user_12345678-f54e-4e09-848c-1953af6e3e89", "pulsar_12345678-6b36-4af2-be1f-d97862c0c41c", $topic, $operation), $topic.starts_with("my-own-prefix");
+
+== Revocation ids ==
+Content-based: 8eaaa639d5b94c3e053ad840e8dcca6fe66a621442ecc99eadf1df03d6138f1d
+Unique:        f608dc2f724fc14faf0daf50774ef0b9425cda26f56ee93317ca80ca13736027
+
+==========
+```
+
+Now the block n°2 ensure that topics must starts with `"my-own-prefix"`.
+
+You can find more examples on the [biscuit-pulsar authorization java tests](https://github.com/CleverCloud/biscuit-pulsar/blob/master/src/test/java/com/clevercloud/biscuitpulsar/AuthorizationProviderBiscuitTest.java).
+
+## Usage
+
+We advise you to use [`pulsarctl`](https://github.com/streamnative/pulsarctl) provided by StreamNative. Here is an example to list topics in your add-on (in your namespace):
+
+```bash
+pulsarctl --admin-service-url $ADDON_PULSAR_HTTP_URL \\
+          --auth-params $ADDON_PULSAR_TOKEN \\
+          --auth-plugin org.apache.pulsar.client.impl.auth.AuthenticationToken \\
+          topics list $ADDON_PULSAR_TENANT/$ADDON_PULSAR_NAMESPACE
+```
+
+As Biscuit is a token, you can use `AuthenticationToken($ADDON_PULSAR_TOKEN)` provided by [clients libraries](https://pulsar.apache.org/docs/en/client-libraries/) to authenticate to our clusters without any tweak.
```

Comment:
I think this should go at the end of the `## Authorization` section, before the `### Attenuation` one